packages/components/header/_header.scss

```diff
@@ -449,8 +449,18 @@
 }
 
 .nhsuk-navigation {
+  /* These negative margins are the same as the padding on the navigation items,
+     and used to align the text of the first and last item with the rest of the header. */
+  margin-left: -16px;
+  margin-right: -16px;
   display: flex;
 
+  @include mq($from: tablet) {
+    /* These padding is slightly more from tablet width upwards. */
+    margin-left: -16px;
+    margin-right: -16px;
+  }
+
```

Comment:
Is this query and these rules needed, given the inline margin for .nhsuk-navigation is already -16px?

```suggestion
```

Reply:
Well spotted - this is no longer needed. I think at one point they were different, but now the margin for tablet upwards is being set on `.nhsuk-header__navigation-list` below instead.